kmultiversx/src/kmultiversx/kdist/mx-semantics/esdt.md

```diff
@@ -147,15 +144,22 @@ module ESDT
           </esdtData>)
           ...
         </account>
-        <instrs> (#waitCommands ~> _) #Or .K </instrs>
+        // <instrs> (#waitCommands ~> _) #Or .K </instrs>
       [priority(61)]
 
-    rule [addToESDTBalance-new-err]:
```

Comment:
If any of these rule names that you change are on one of these lists, please update them in a follow-up PR:
https://github.com/runtimeverification/mx-backend/blob/master/kmxwasm/src/kmxwasm/property_testing/running.py#L35-L71